### PR TITLE
Updated registration email templates.

### PR DESCRIPTION
**Pull request by aspidites**
Fixed the fact that my templates were being overridden.

Password Reset:
2015-01-05-111209_1465x304_scrot

Activation:
2015-01-05-111428_1432x239_scrot

### CODE REVIEW

**Comment by aspidites** [comment]
For the record, this is what happens if I try to use underscores as an hr (since django password reset emails are plain text:
2015-01-05-111734_1482x349_scrot

File: registration/templatetags/password_reset_tags.py [hunk 1]
```diff
@@ -1,17 +1,37 @@
 from django.template import Library
 from django.conf import settings
 
+from seo.models import SeoSite
+
 register = Library()
 
 
 @register.simple_tag
-def get_current_seosite_domain():
```
Comment (tdphillips) [inline registration/templatetags/password_reset_tags.py]: /templates/postajob/request_email.html still uses this name.

Comment (aspidites) [inline registration/templatetags/password_reset_tags.py]: Fixed

**Comment by aspidites** [comment]
With dashes:
2015-01-05-130129_1447x340_scrot

@fezick ?

**Comment by fezick** [comment]
:+1:

**Pull request merged**